src/components/ha-code-editor.ts

```diff
@@ -0,0 +1,129 @@
+// @ts-ignore
+import CodeMirror from "codemirror";
```

Comment:
I think that we should async load both this and the CSS (as 1 module with 1 `import()`).